src/display/canvas.js

```diff
@@ -1698,6 +1699,14 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       this.current.fillColor = color;
       this.current.patternFill = false;
     },
+    setStrokeCMYKColor: function CanvasGraphics_setStrokeCMYKColor(c,
+                                                    m, y, k, r, g, b) {
+      this.setStrokeRGBColor(r, g, b);
```

Comment:
This, and the similar changes below, will add run-time overhead (two function calls rather than just one) to the default PDF.js use-case despite this not really being used for anything, which overall doesn't seem great.

Reply:
Is code duplication prefered to duplicate calls?

```js
setStrokeCMYKColor: function CanvasGraphics_setStrokeCMYKColor(c,
                                                    m, y, k, r, g, b) {
      var color = Util.makeCssRgb(r, g, b);
      this.ctx.fillStyle = color;
      this.current.fillColor = color;
      this.current.patternFill = false;
}
```

Reply:
Generally you'd want to avoid code-duplication as well, especially when it's essentially unused by default.

Reply:
I would reason the same. But I didnt think that the overhead for two function calls in a row had any significant impact on the overall performance in the application (as the solution is implemented right now).

Reply:
Reducing the total number of function calls can add-up in certain cases, and even if this particular instance perhaps isn't too hot, it still doesn't seem necessary when it's (again) essentially unused.